app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -226,49 +226,50 @@ public String getText() {
     }
 
     public static class ChatHolder extends RecyclerView.ViewHolder {
-        View mView;
+        private final TextView mNameField;
+        private final TextView mTextField;
+        private final FrameLayout mLeftArrow;
+        private final FrameLayout mRightArrow;
+        private final RelativeLayout mMessageContainer;
+        private final LinearLayout mMessage;
 
         public ChatHolder(View itemView) {
             super(itemView);
-            mView = itemView;
+            mNameField = (TextView) itemView.findViewById(R.id.name_text);
+            mTextField = (TextView) itemView.findViewById(R.id.message_text);
+            mLeftArrow = (FrameLayout) itemView.findViewById(R.id.left_arrow);
+            mRightArrow = (FrameLayout) itemView.findViewById(R.id.right_arrow);
+            mMessageContainer = (RelativeLayout) itemView.findViewById(R.id.message_container);
+            mMessage = (LinearLayout) itemView.findViewById(R.id.message);
         }
 
-        public void setIsSender(Boolean isSender) {
-            FrameLayout left_arrow = (FrameLayout) mView.findViewById(R.id.left_arrow);
-            FrameLayout right_arrow = (FrameLayout) mView.findViewById(R.id.right_arrow);
-            RelativeLayout messageContainer = (RelativeLayout) mView.findViewById(R.id.message_container);
-            LinearLayout message = (LinearLayout) mView.findViewById(R.id.message);
-
-            int color;
+        public void setIsSender(boolean isSender) {
+            final int color;
             if (isSender) {
-                color = ContextCompat.getColor(mView.getContext(), R.color.material_green_300);
-
-                left_arrow.setVisibility(View.GONE);
-                right_arrow.setVisibility(View.VISIBLE);
-                messageContainer.setGravity(Gravity.END);
+                color = ContextCompat.getColor(itemView.getContext(), R.color.material_green_300);
```

Comment:
Considered that, but those could actually theoretically change when the configuration changes as well.  If we want to assume they don't, then they're cachable as well.